src/collections.rs

```diff
@@ -2,7 +2,11 @@
 
 use crate::alloc::{Layout, LayoutErr};
 use core::fmt::Display;
-pub use liballoc::collections::{binary_heap, btree_map, btree_set, linked_list, vec_deque};
+pub use liballoc::collections::{binary_heap, btree_set, linked_list, vec_deque};
+
+pub mod btree_map {
+    pub use super::super::btree::map::*;
```

Comment:
Better use an absolute path. `super::super::` is really verbose and hard to read.